src/pages/Checkout/CheckoutPage.vue

```diff
@@ -163,19 +163,22 @@
 		</div>
 		<div v-if="emptyBasket" class="empty-basket">
 			<div class="row display-align text-center">
-				<h2 class="empty-basket-heading">Your basket is empty!</h2>
-				<p>But we'd love to help you change that! Please consider
-				supporting one of the borrowers below, or
-					<a href="/lend-by-category">browse all loans</a>.
-				</p>
+				<div class="columns small-12">
```

Comment:
wrapped in column to ensure right and left margins. previously text was up against edge of page